samples/net/sockets/civetweb_websocket_server/src/libc_extensions.c

```diff
@@ -0,0 +1,250 @@
+/*
```

Comment:
BTW, the libc_extensions.c file and the *.inl files are probably the same between various civetweb based applications. So should we re-use them between app instead of replicating the code.
I was thinking directory structure like (in the example below, the http_server would be the current civetweb sample application):
```
samples/net/sockets/civetweb/
	lib/
	http_server/
	websocket_server/
	....
```
What do you think?

Reply:
It is a good idea. I thought about it already. My opinion here is: First push all my current civetweb dependent PRs as they currently are, then putting it in separate civetweb folder with possible optimisations. I think, this will be possible until v2.5 release. Moreover, all the libc/minlibc functionalities may be changed with updated posix libc implementations.

Reply:
Great, sounds like a plan!